modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/AdminTooltip.js

```diff
@@ -0,0 +1,240 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import ClayAutocomplete from '@clayui/autocomplete';
+import ClayButton from '@clayui/button';
+import ClayCard from '@clayui/card';
+import ClayDropDown from '@clayui/drop-down';
+import ClayForm, {ClayInput, ClayRadio, ClayRadioGroup} from '@clayui/form';
+import PropTypes from 'prop-types';
+import React, {useEffect, useState} from 'react';
+
+const AdminTooltip = ({
+	deletePin,
+	namespace,
+	searchSkus,
+	setRemovePinHandler,
+	setShowTooltip,
+	showTooltip,
+	skus,
+	updatePin,
+}) => {
+	const [pinPositionLabel, setPinPositionLabel] = useState(
+		showTooltip.details.label
+	);
+	const [linkedValue, setLinkedValue] = useState(
+		showTooltip.details.linked_to_sku
+	);
+	const [sku, setSku] = useState('' || showTooltip.details.sku);
+	const [quantity, setQuantity] = useState(showTooltip.details.quantity);
+
+	useEffect(() => {
+		searchSkus(sku);
+	}, [sku, searchSkus]);
+
+	return (
+		<ClayCard
+			className="admin-tooltip"
+			style={{
+				left: showTooltip.details.cx,
+				top: showTooltip.details.cy,
+			}}
+		>
+			<ClayCard.Body className="row">
+				<ClayForm.Group className="col-12 form-group-sm text-left">
```

Comment:
You can use the `small` prop for form-group-small. [see api](https://clayui.com/docs/components/form/api.html)

Reply:
done